PR_TESTER.jenkinsfile

```diff
@@ -0,0 +1,98 @@
+@Library('migrationqe-automation-lib') _
+pipeline{
+    agent {
+        label 'minikube-node' 
+    }
+    options{
+        ansiColor('xterm')
+    } 
+    environment{
+        HUB_USERNAME="admin"
+        PATH="$PATH:/usr/local/go/bin"
+        HUB_PASSWORD="Passw0rd!"
```

Comment:
Assuming this is some example password, not working on some running system.

Reply:
It is the default password and it is used with `keycloak_sso_req_passwd_update: false`

Reply:
I don't think it is the right thing to store even default passwords in public git repo, but just my $0.02.

Reply:
@aufi Thanks for your input , I don't mind removing these from here. I'll make the changes :)